src/smartpeak/source/core/RawDataProcessor.cpp

```diff
@@ -148,7 +148,7 @@ namespace SmartPeak
       else {
         OpenMS::FileHandler fh;
         LOGI << "Loading: " << filenames.mzML_i;
-        fh.loadExperiment(filenames.mzML_i, chromatograms);
+        fh.loadExperiment(filenames.mzML_i, chromatograms); //TODO:try-catch (SIGABRT)
```

Comment:
What do you mean by this comment?

Reply:
Occasionally, the `OpenMS::Internal::XMLFile::parse_` fails to parse a given file due to `QDateTime::fromString` being thread-unsafe, it causes an instant crash on macOS and needs to be looked into but is not part of the PR.